.github/workflows/astarte-ctl-test.yaml

```diff
@@ -139,7 +139,11 @@ jobs:
 
       - name: Install dependencies    
         shell: bash
-        run: pip install -e ./tests 
+        run: pip install -e ./tests
+        
+      - name: Format check
+        shell: bash
+        run: python -m black -l 100 --diff --check tests/ 
```

Comment:
In `pyproject.toml` [file](https://github.com/astarte-platform/astartectl_test_suite/blob/a3fa8db8ce09fddb25598cc1b952520d979ba257/tests/pyproject.toml#L26) we defined line length to be 88 columns. Either change here from 100 to 88 or change `pyproject.toml` specification to 100 or workflow check to 88. I would rather change `pyproject.toml` to 88 columns.

Reply:
What do you think about setting it to 100 columns? I have some long lines of code.

Reply:
Of course. Fine with me.